velox/exec/tests/PartitionedOutputBufferManagerTest.cpp

```diff
@@ -38,7 +38,7 @@ class PartitionedOutputBufferManagerTest : public testing::Test {
 
   void SetUp() override {
     pool_ = facebook::velox::memory::addDefaultLeafMemoryPool();
-    bufferManager_ = PartitionedOutputBufferManager::getInstance().lock();
+    bufferManager_ = OutputBufferManager::getInstance().lock();
```

Comment:
Should we rename the test as well? 

PartitionedOutputBufferManagerTest.cpp -> OutputBufferManagerTest.cpp